Error: Cannot find module 'firebase/auth'

after fixing the npm dev issue, I was able to run the dev server and this occured.

<img width="215" alt="Screenshot 2024-07-13 105527" src="https://github.com/user-attachments/assets/fd66427b-75a0-4ed4-ab30-4dddee1f80e5">

```powershell
 *  Executing task: npm run dev 


> untitled-princess-carolyn-project@0.0.1 dev
> cross-env NODE_ENV=development vite


  VITE v5.3.1  ready in 1169 ms

  ➜  Local:   http://localhost:5173/
  ➜  Network: use --host to expose
  ➜  press h + enter to show help
10:52:12 AM [vite] Pre-transform error: Failed to load url firebase/auth (resolved id: firebase/auth) in C:/Users/Len/Documents/Thirdplace project/thirdplace-mvp/src/lib/utils/auth.ts. Does the file exist?
10:52:12 AM [vite] Pre-transform error: Failed to load url firebase/auth (resolved id: firebase/auth) in C:/Users/Len/Documents/Thirdplace project/thirdplace-mvp/src/lib/utils/auth.ts. Does the file exist? (x2)
10:52:12 AM [vite] Pre-transform error: Failed to load url firebase/app (resolved id: firebase/app) in C:/Users/Len/Documents/Thirdplace project/thirdplace-mvp/src/lib/utils/firebaseSetup.ts. Does the file exist?
10:52:12 AM [vite] Error when evaluating SSR module /src/lib/utils/auth.ts: failed to import "firebase/auth"   
|- Error: Cannot find module 'firebase/auth' imported from 'C:/Users/Len/Documents/Thirdplace project/thirdplace-mvp/src/lib/utils/auth.ts'
    at nodeImport (file:///C:/Users/Len/Documents/Thirdplace%20project/thirdplace-mvp/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53484:19)
    at ssrImport (file:///C:/Users/Len/Documents/Thirdplace%20project/thirdplace-mvp/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53349:22)
    at eval (eval at instantiateModule (file:///C:/Users/Len/Documents/Thirdplace%20project/thirdplace-mvp/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53398:24), <anonymous>:6:37)
    at async instantiateModule (file:///C:/Users/Len/Documents/Thirdplace%20project/thirdplace-mvp/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53408:5)

10:52:12 AM [vite] Error when evaluating SSR module /src/routes/+page.svelte: failed to import "firebase/auth" 
|- Error: Cannot find module 'firebase/auth' imported from 'C:/Users/Len/Documents/Thirdplace project/thirdplace-mvp/src/lib/utils/auth.ts'
    at nodeImport (file:///C:/Users/Len/Documents/Thirdplace%20project/thirdplace-mvp/node_modules/vite/dist/node/chde/chunks/dep-BcXSligG.js:53484:19)
    at ssrImport (file:///C:/Users/Len/Documents/Thirdplace%20project/thirdplace-mvp/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53349:22)
    at eval (eval at instantiateModule (file:///C:/Users/Len/Documents/Thirdplace%20project/thirdplace-mvp/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53398:24), <anonymous>:6:37)
    at async instantiateModule (file:///C:/Users/Len/Documents/Thirdplace%20project/thirdplace-mvp/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53408:5)

Error: Cannot find module 'firebase/auth' imported from 'C:/Users/Len/Documents/Thirdplace project/thirdplace-mvp/src/lib/utils/auth.ts'
    at nodeImport (file:///C:/Users/Len/Documents/Thirdplace%20project/thirdplace-mvp/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53484:19)
    at ssrImport (file:///C:/Users/Len/Documents/Thirdplace%20project/thirdplace-mvp/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53349:22)
    at eval (eval at instantiateModule (file:///C:/Users/Len/Documents/Thirdplace%20project/thirdplace-mvp/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53398:24), <anonymous>:6:37)
    at async instantiateModule (file:///C:/Users/Len/Documents/Thirdplace%20project/thirdplace-mvp/node_modules/vite/dist/node/chunks/dep-BcXSligG.js:53408:5) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

--- a/package.json
+++ b/package.json
@@ -25,6 +25,7 @@
 	},
 	"type": "module",
 	"dependencies": {
+		"firebase": "^10.12.3",
 		"firebase-admin": "^12.2.0",
 		"svelte-preprocess": "^6.0.1"
 	}

--- a/package.json
+++ b/package.json
@@ -3,7 +3,7 @@
 	"version": "0.0.1",
 	"private": true,
 	"scripts": {
-		"dev": "NODE_ENV=development vite",
+		"dev": "cross-env NODE_ENV=development vite",
 		"build": "vite build",
 		"preview": "vite preview",
 		"check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
@@ -14,6 +14,7 @@
 		"@sveltejs/kit": "^2.0.0",
 		"@sveltejs/vite-plugin-svelte": "^3.0.0",
 		"autoprefixer": "^10.4.19",
+		"cross-env": "^7.0.3",
 		"postcss": "^8.4.38",
 		"svelte": "^4.2.18",
 		"svelte-check": "^3.6.0",

